src/services/microfrontends-service.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from 'node:path'
+import * as fs from 'node:fs'
+import { CLIError } from '@oclif/errors'
+import { BundleDescriptor, MicroFrontend } from '../models/bundle-descriptor'
+import BundleDescriptorService from './bundle-descriptor-service'
+
+const MICROFRONTENDS_DIRNAME = 'microfrontends'
+
+export class MicroFrontendsService {
+  private readonly microfrontendsPath: string
+  private readonly bundleDescriptorService: BundleDescriptorService
+
+  constructor() {
+    this.microfrontendsPath = path.resolve(
+      process.cwd(),
+      MICROFRONTENDS_DIRNAME
+    )
+    this.bundleDescriptorService = new BundleDescriptorService(process.cwd())
+  }
+
+  public addMicroFrontend(mfe: MicroFrontend): void {
```

Comment:
Let's perform some sanitization of the mfe name (use the same regex used in initializer-service) in order to avoid bad chars in the mfe name